### ISSUE TITLE
Syntax Error when using Laravel Enum Validation Rule

### ISSUE THREAD

**Issue by geneowak** [prettier/plugin-php#1955]
**Prettier 2.6.1**

**PHP Plugin 0.18.4**

```sh
# Options (if any):
{
  "printWidth": 80,
  "tabWidth": 4,
  "useTabs": false,
  "singleQuote": true,
  "phpVersion": "8.1",
  "trailingCommaPHP": true,
  "braceStyle": "psr-2",
  "requirePragma": false,
  "insertPragma": false
}
```

**Input:**

```php
<?php

namespace App\Traits;

use App\Enums\PrefferedNotificationType;
use Illuminate\Validation\Rules\Enum;

trait SomeValidationRules
{
    public function creationRules()
    {
        return [
            'preffered_notification_type' => ['required', new Enum(PrefferedNotificationType::class)],
        ];
    }
}
```

**Output:**

```php
Parse Error : syntax error, unexpected 'Enum' (T_ENUM), expecting ';' on line 6
```

**Expected behavior:**

No error

**Comment by Berbass** [comment]
Probably related to https://github.com/prettier/plugin-php/issues/1942

**Comment by elev8studio** [comment]
Is there any update on this? I'm experiencing the same issue.

**Comment by czosel** [comment]
Fixed in `v0.18.5` :tada:
